phpseclib/Crypt/RSA.php

```diff
@@ -2072,55 +2055,133 @@ function _rsassa_pkcs1_v1_5_verify($m, $s)
     }
 
     /**
-     * Set Encryption Mode
+     * RSASSA-PKCS1-V1_5-VERIFY (relaxed matching)
      *
-     * Valid values include self::ENCRYPTION_OAEP and self::ENCRYPTION_PKCS1.
+     * Per {@link http://tools.ietf.org/html/rfc3447#page-43 RFC3447#page-43} PKCS1 v1.5
+     * specified the use BER encoding rather than DER encoding that PKCS1 v2.0 specified.
+     * This means that under rare conditions you can have a perfectly valid v1.5 signature
+     * that fails to validate with _rsassa_pkcs1_v1_5_verify(). PKCS1 v2.1 also recommends
+     * that if you're going to validate these types of signatures you "should indicate
+     * whether the underlying BER encoding is a DER encoding and hence whether the signature
+     * is valid with respect to the specification given in [PKCS1 v2.0+]". so if you do
+     * $rsa->getLastPadding() and get RSA::PADDING_RELAXED_PKCS1 back instead of
+     * RSA::PADDING_PKCS1... that means BER encoding was used.
      *
-     * @access public
-     * @param int $mode
+     * @access private
+     * @param string $m
+     * @return bool|string
```

Comment:
Should this be `@return bool`? Because all returns, including `$this->_equals($em, $em2)`, are of boolean type.